S2VX.Game/Editor/ToolState/SelectToolState.cs

```diff
@@ -173,26 +177,27 @@ public class SelectToolState : S2VXToolState {
             return gameTime + (leftOffset <= rightOffset ? -leftOffset : rightOffset);
         }
 
+        private double GetGameTimeAtMouse(Vector2 localMousePos) {
+            var tickBarNoteSelections = Editor.NotesTimeline.TickBarNoteSelections;
+            var mousePosX = ToSpaceOfOtherDrawable(ToLocalSpace(localMousePos), tickBarNoteSelections).X;
+            // temp until NoteTimeline Scroll on drag is implemented
```

Comment:
What does this comment mean?

Reply:
Osu!'s NoteTimeline scrolls when a note is dragged to the edges. I added a card in Tasks a few hours ago so I can maybe handle this in the future. Right now I'm just clamping mousePox within the NoteTimeline.

Reply:
Sounds good